src/conf_mode/service_ssh.py

```diff
@@ -95,6 +119,22 @@ def generate(ssh):
         syslog(LOG_INFO, 'SSH ed25519 host key not found, generating new key!')
         call(f'ssh-keygen -q -N "" -t ed25519 -f {key_ed25519}')
 
+    if 'trusted_user_ca_key' in ssh:
```

Comment:
```suggestion
    if 'trusted_user_ca_key' in ssh:
        ...
    elif os.path.exists(trusted_user_ca_key):
        os.unlink(trusted_user_ca_key)
```

Even though the config line is removed, best to cleanup the unused file that would otherwise persist in /etc

Reply:
hi @sarthurdev san!
Thank you for continuing the review!
I will accept the suggestions as they are.
I will only modify the commit message.